assets/styles/elements/menu-item-link.css

```diff
@@ -0,0 +1,28 @@
+.menu-item__link {
+    all: unset;
+    width: 16rem;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
+    height: 100%;
+    gap: 2rem;
+    padding: 1rem 0.5rem;
+    box-sizing: border-box;
+    transition:
+        box-shadow 0.3s ease,
+        color 0.3s ease;
+    border: 0.0625rem solid transparent;
+}
+
+.menu-item__link:hover {
+    box-shadow: 0 0 0 0.0625rem var(--color-primary);
+}
+
+.menu-item__link svg path {
```

Comment:
There is an alternative way of styling an SVG's colors without limitation to specific elements. In the SVG we can set `stroke="currentColor"` (works also for `fill`). With `currentColor` the font color will be used.

Then we can do something like this:

```css

.menu-item__icon {
    transition: color 0.3s ease;
}

.menu-item__icon:hover {
    color: var(--color-primary);
}
```

Where the icon must be wrapped into a styling element:

```html
<div class="menu-item__icon">
    {{- partial "graphics/vector.html" -}}
</div>
```